src/components/JobWizard/JobWizard.js

```diff
@@ -402,6 +402,11 @@ const JobWizard = ({
 
       return [
         {
+          id: isBatchInference
+            ? 'job-wizard-btn-schedule-infer'
+            : isTrain
+            ? 'job-wizard-btn-schedule-training'
+            : 'job-wizard-btn-schedule-later',
```

Comment:
There is no need to add different IDs for the single button. You can have just `schedule-btn`